math/convertall/Portfile

```
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           qt5 1.0
+
+github.setup        doug-101 ConvertAll 0.8.0 v
```

```suggestion
github.setup        doug-101 convertall 0.8.0 v
```

This then allows us to remove the `name` attribute.